keyboards/mysticworks/wyvern/config.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2020 Albert Dong <albert@mysticworks.xyz>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device Descriptor Parameter */
+#define VENDOR_ID       0x4D57
+#define PRODUCT_ID      0x0001
```

Comment:
This might be better. 
```suggestion
#define PRODUCT_ID      0x39E8
```

Reply:
Could you explain why this would be better?

Reply:
If this is about the Vendor ID not being unique, just pushed a change to fix that.